desktop-exporter/types.go

```diff
@@ -0,0 +1,71 @@
+package desktopexporter
```

Comment:
Something about having types laid out all organized like this makes me very happy